common/src/chain/transaction/account_outpoint.rs

```diff
@@ -110,6 +115,10 @@ pub enum AccountCommand {
     // Change the authority who can authorize operations for a token
     #[codec(index = 5)]
     ChangeTokenAuthority(TokenId, Destination),
+    #[codec(index = 6)]
+    WithdrawOrder(OrderId),
```

Comment:
Please call it CancelOrder

Reply:
Please make sure to grep the whole code and change every withdraw to cancel... I can imagine comments having it. Be aware that the word "withdraw" is used for delegations.